infrastructure/export.py

```diff
@@ -106,6 +107,24 @@ def refresh_views():
                 ON l.organization_id = r.id
                 GROUP BY l.project_id;
             ''')
+        cursor.execute('''
+            CREATE OR REPLACE VIEW infrastructure_power_plant_owners_view AS
+                SELECT l.id, 
+                    array_to_string(array_agg(quote_literal(r."name")), ', ', 'NULL') AS owners
+                FROM infrastructure_powerplant_owners AS l
```

Comment:
We want to use ``infrastructure_ownerstake`` here, not ``infrastructure_powerplant_owners`` (we're going to delete that column in a future ticket).